lib/inspector.coffee

```diff
@@ -50,7 +56,7 @@ class Inspector
             cursor_pos = cursor.getBufferColumn()
         return [code, cursor_pos]
 
-    getInspector: ->
+    initializeInspector: ->
```

Comment:
This method is only used by the constructor. Can it be moved to the constructor?
And I'd rename `@inspector` to `@view`.